www/source/docs/concepts-depot.html.md

```diff
@@ -6,6 +6,10 @@ title: Depot
 
 A service for storing Habitat packages for download and upload.
 
+Private depots are in the works but currently only the public depot is supported. You may explore the public depot via its web interface at [https://app.habitat.sh](https://app.habitat.sh)
+
+In addition to viewing packages, you may use the origins pane to manage your origins, optionally uploading your public org key via the webui rather than the `hab origin key upload` command. 
```

Comment:
I'll defer to others on this, but considering extensive changes coming to depot in a matter of a couple days we might want to hold off on this. Theres likely some better documentation coming here. (I could be wrong though :) )